Kudu.SiteManagement/SiteManager.cs

```diff
@@ -347,6 +345,57 @@ public bool RemoveSiteBinding(string applicationName, string siteBinding, SiteTy
             }
         }
 
+        public async Task StartAppPool(string applicationName)
+        {
+            using (ServerManager iis = GetServerManager())
+            {
+                string appPoolName = GetAppPool(applicationName);
+                var kuduAppPool = iis.ApplicationPools[appPoolName];
+
+                if (kuduAppPool.State != ObjectState.Started)
```

Comment:
will there be a case that we try to start the app pool while it is in "Starting" status? if so, will it work?